Sources/SmithyChecksums/SigningConfig+Extension.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+extension SigningConfig {
```

Comment:
moved these methods out of AwsChunkedUtils in AWSSDKChecksums as they are used in smithy-swift

Reply:
Does this mean that anything that is signed with SigV4 requires `SmithyChecksums`?

Reply:
As of now SigV4 depends on only `AWSSDKChecksums` but AWSSDKChecksums depends on `SmithyChecksums` because the `setChunkedBody()` method creates ` ChunkedStream`.